libs/src/services/creatorNode/index.js

```diff
@@ -2,7 +2,7 @@ const axios = require('axios')
 const FormData = require('form-data')
 const retry = require('async-retry')
 
-const { wait } = require('../../utils')
+const { wait } = require('../../utils').Utils
```

Comment:
can we actually destructure Utils like this?
```
const { Utils: { wait } } = require('../../utils')

```

Reply:
changed